packages/react-graphql/src/hooks/README.md

```diff
@@ -0,0 +1,94 @@
+# Using Apollo Hooks
+
+## `ApolloProvider`
+
+Before using the individual hooks, you will need to wrap your application with `ApolloProvider` at root of your React component tree.
+
+You can replace `react-apollo`'s [`ApolloProvider`](https://www.apollographql.com/docs/react/api/react-apollo#ApolloProvider) if you are only going to use Hooks.
```

Comment:
Should ours just render the "real" `ApolloProvider` so that you only need one no matter what?